docs/symptom-survey/contingency-tables.md

```diff
@@ -168,321 +166,18 @@ indicator columns will be marked as missing (`NA`) for that group.
 
 ## Indicators
 
+<div style="background-color:#f5f6fa; padding: 10px 30px;"><strong>Indicator
+codebook:</strong> Our <a href="contingency-codebook.csv">contingency table
```

Comment:
Linking directly to the Google spreadsheet version will make it easier to keep this up to date. We can set it up so it auto-downloads as a CSV from the link, if you'd like to keep that format.